### PR TITLE
Book UI

### LINKED ISSUE #5: Book UI

### PR DESCRIPTION
**Pull request by chantlong**
Webcam Preview Button Works on Side Menu
Webcam Invite Menu Pops Up on Click
Webcam Invite Menu closes upon successful load
Resolve #5 

screen shot 2016-06-09 at 22 58 58

### CODE REVIEW

**Comment by chantlong** [comment]
The socket io only works on this URL
http://localhost:3000/story-time

If done with the UNIQ BOOK ID URL, it fails.

File: client/index.html [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <script type="text/javascript" src="https://code.jquery.com/jquery-1.7.1.min.js"></script>
   <script src="https://media.twiliocdn.com/sdk/js/common/v0.1/twilio-common.min.js"></script>
   <script src="https://media.twiliocdn.com/sdk/js/conversations/v0.13/twilio-conversations.min.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/css/materialize.min.css">
```
Comment (mvargeson) [inline client/index.html]: can probably be removed right?

File: client/styles/style.css [hunk 2]
```diff
@@ -47,97 +76,102 @@ body.story-time {
...
+  font-size: 25px;
+  z-index: 1;
+  width: 430px;
+  height: 300px;
+  position: absolute;
+  top: 190px;
+  left: 740px;
+}
+
+/*.left-page-image {
```
Comment (mvargeson) [inline client/styles/style.css]: remove commented code

File: client/styles/style.css [hunk 3]
```diff
@@ -197,26 +225,26 @@ input {
...
-  font-size: 18px;
+  font-family: Avenir, sans-serif;  
+  font-size: 14px;
   text-align: center;
 }
 
-:-moz-placeholder { /* Firefox 18- */
-  font-family: TravelingTypewriter, sans-serif;  
-  font-size: 18px;
+/*:-moz-placeholder*/ { /* Firefox 18- */
```
Comment (mvargeson) [inline client/styles/style.css]: ?

File: client/styles/style.css [hunk 4]
```diff
@@ -197,26 +225,26 @@ input {
...
+  font-family: Avenir, sans-serif;  
+  font-size: 14px;
   text-align: center;
 }
 
-::-moz-placeholder {  /* Firefox 19+ */
-  font-family: TravelingTypewriter, sans-serif;  
-  font-size: 18px;
+/*::-moz-placeholder { */ /* Firefox 19+ */
```
Comment (mvargeson) [inline client/styles/style.css]: ?

File: server/controllers/books.js [hunk 5]
```diff
@@ -15,7 +15,7 @@ module.exports = {
       bookData: [
         {
           name: 'Page1',
-          content: 'In the light of the moon a little egg lay on a leaf.',
+          content: 'In the light of the moon a little egg lay on a leaf. I ate a banana and tripped on it like a boss',
```
Comment (mvargeson) [inline server/controllers/books.js:18]: ... like a boss

**Pull request merged**